server/sentence_boundary.py

```
@@ -0,0 +1,69 @@
+                # Yield and clear the current sentence
+
+                yield remove_asterisks(self.current_sentence.strip())
+                processed_chunks.append(self.current_sentence)  # Add processed chunk
+                self.current_sentence = match_text
+            if not ABBREVIATION_RE.search(self.current_sentence[-5:]):
+                yield remove_asterisks(self.current_sentence.strip())
+                processed_chunks.append(self.current_sentence)  # Add processed chunk
+                self.current_sentence = ""
+            # Update the remaining text
+
+            self.remaining_text = self.remaining_text[match.end() :]
```

Flake8 (7.2.0)

[error] whitespace before ':'

(E203)